docs/architecture/architecture-overview.adoc

```diff
@@ -0,0 +1,244 @@
+
+image::images/5.x-ServiceCatalog-MobileServices.png[width=1280, title="Service Catalog - Mobile Service Perspective", link="images/5.x-ServiceCatalog-MobileServices.png"]
+
+What happens behind the scene is that the 'provision' ansible playbook of the service in question is invoked. This does the provisioning of the service into the project.
+
+As part of this provisioning two objects are also created, a configMap and a secret for the new service in OpenShift, see Figure below.
+
+image::images/5.x-Architecture-MC-ServiceInstantiation.png[width=1280, title="Instantiated/Provisioned Mobile Service", link="images/5.x-Architecture-MC-ServiceInstantiation.png"]
+
+The ConfigMap which is created contains public info about the mobile service. The info conatined in this ConfigMap is necessary for a remote/real Mobile Client/App to be able to interact with and consume the Mobile Service. It will be most likely be the Core Client SDK which will provide the Mobile Client with this configuration (TBC). The Secret that is created holds private info about the service, e.g. uname and pwd.
+
+=== Integrate two Mobile Services
+The MC provides an initiation point where Mobile Services can be integrated with each other. Once there are Mobile Services provisioned, potential bindings (integrations with other Mobile Services) available for each Mobile Service is read by MC and is presented as potential integration options for each service. The 'integrations' field in the metadata, identifies what integrations/bindings can be created for the service in question.
+
+. Extract from the https://github.com/aerogearcatalog/fh-sync-server-apb/blob/master/apb.yml[Synch Mobile Service APB]
+====
+ metadata:
+  displayName: FeedHenry Sync Server
+  serviceName: fh-sync-server
+  integrations: keycloak,apiKeys
+====
+
+
+When a developer decides to perform an integration between two Mobile Services, via the UI or through the CLI, e.g.
+
+ > mobile create integration <consuming_service_instance_id> <providing_service_instance_id> --namespace=<yourProjectName>
+
+a number of actions are taken:
+
+* the bind APB is ran for providing service
+** Example of https://github.com/aerogearcatalog/keycloak-apb/blob/master/roles/bind-keycloak-apb/tasks/main.yml[Keycloak Bind APB]
+* the MC would create a PodPreset for the consuming service.
+** A https://docs.openshift.com/container-platform/3.6/dev_guide/pod_preset.html[pod preset] is an object that injects user-specified information into pods as they are created.
+** The MC uses the pod preset concept to inject the secret created by the binding of the providing service into the consuming service (e.g. if you wanted to protect the Mobile Synch Service with authentication from Keycloak, the Keycloak secret would be injected into the Synch Service)
```

Comment:
The MC doesn't do anything here.
Maybe something like this would read better?
"The Pod Preset defines how the Secret should be mounted into the consuming service"